google/cloud/bigquery/schema.py

```diff
@@ -86,9 +98,22 @@ def __init__(
             self._properties["mode"] = mode.upper()
         if description is not _DEFAULT_VALUE:
             self._properties["description"] = description
+        if precision is not _DEFAULT_VALUE:
+            self._properties["precision"] = precision
+        if scale is not _DEFAULT_VALUE:
+            self._properties["scale"] = scale
+        if maxLength is not _DEFAULT_VALUE:
+            self._properties["maxLength"] = maxLength
         self._fields = tuple(fields)
         self._policy_tags = policy_tags
 
+    @staticmethod
+    def __get_int(f, name):
+        v = f.get(name, _DEFAULT_VALUE)
```

Comment:
It might not be immediately clear what `f` means, let's to rename to `field`? Or `field_dict`?

I actually expected the linter to complain here ([naming style guide](https://google.github.io/styleguide/pyguide.html#s3.16-naming)), but I guess that was on a different project. :)

Reply:
done